CMakeLists.txt

```diff
@@ -342,6 +348,8 @@ if (USE_ROS)
       set(OROCOS_KDL_LIBRARIES ${ARM_CHROOT_DIR}/opt/ros/kinetic/lib/liborocos-kdl.so)
     elseif("${ROS_DISTRO}" STREQUAL "melodic")
       set(OROCOS_KDL_LIBRARIES ${ARM_CHROOT_DIR}/opt/ros/melodic/lib/liborocos-kdl.so)
+    elseif("${ROS_DISTRO}" STREQUAL "noetic")
```

Comment:
actually, maybe for all of these you can set a $ROS_DIR to simplify things greatly :)